databuilder/databuilder/models/table_stats.py

```diff
@@ -10,20 +10,128 @@
 from databuilder.models.graph_node import GraphNode
 from databuilder.models.graph_relationship import GraphRelationship
 from databuilder.models.graph_serializable import GraphSerializable
-from databuilder.models.table_metadata import ColumnMetadata
+from databuilder.models.table_metadata import ColumnMetadata, TableMetadata
 from databuilder.models.table_serializable import TableSerializable
 
+LABEL = 'Stat'
+STAT_RESOURCE_RELATION_TYPE = 'STAT_OF'
+RESOURCE_STAT_RELATION_TYPE = 'STAT'
+
+
+class TableStats(GraphSerializable, TableSerializable):
+    """
+    Table stats model.
+    """
+
+    KEY_FORMAT = '{db}://{cluster}.{schema}' \
+                 '/{table}/{stat_type}/'
+
+    def __init__(self,
+                 table_name: str,
+                 stat_name: str,
+                 stat_val: str,
+                 is_metric: bool,
```

Comment:
could you leave a comment about what this means. "metrics" and "stats" are synonyms generally so we should define what distinction we're making